boards/arm/adafruit_feather_nrf52840/board.cmake

```diff
@@ -2,6 +2,7 @@
 
 board_runner_args(jlink "--device=nRF52840_xxAA" "--speed=4000")
 board_runner_args(pyocd "--target=nrf52840" "--frequency=4000000")
+include(${ZEPHYR_BASE}/boards/common/bossac.board.cmake)
 include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
 include(${ZEPHYR_BASE}/boards/common/pyocd.board.cmake)
 include(${ZEPHYR_BASE}/boards/common/blackmagicprobe.board.cmake)
```

Comment:
264KB image? Please scale that way down to 50KB or so by cropping it and using https://tinypng.com/

Reply:
Done